tests/conftest.py

```diff
@@ -39,8 +36,13 @@ def _mark_masked(obj, path_items):
 
 
 def _clean_output_json(output_json: str) -> str:
-    """Make semgrep's output deterministic and nicer to read."""
-    output = json.loads(output_json)
+    """Make JSON output deterministic and nicer to read."""
+    try:
+        output = json.loads(output_json)
+    except json.JSONDecodeError:
+        raise ValueError(
+            f"Instead of JSON, output was:\n--- output start ---\n{output_json}\n--- output end ---"
```

Comment:
```suggestion
            f"Instead of JSON, Semgrep output was:\n--- output start ---\n{output_json}\n--- output end ---"
```

Reply:
probably also use printing the semgrep stderr in this case somehow?

Reply:
it's printed above and pytest shows captured print output when a test fails